src/stats.ts

```diff
@@ -4,15 +4,15 @@ const PER_PAGE = 100;
 
 function getApp() {
   return new App({
-    appId: process.env.APP_ID,
-    privateKey: process.env.PRIVATE_KEY,
+    appId: process.env.APP_ID ?? "",
```

Comment:
is there ever a time when these values are missing?

Reply:
I just tried to add error handling